contracts/governance/Vesting/fouryear/FourYearVestingLogic.sol

```diff
@@ -108,13 +113,26 @@ contract FourYearVestingLogic is IFourYearVesting, FourYearVestingStorage, Appro
         emit VotesDelegated(msg.sender, _delegatee);
     }
 
+    /**
+     * @notice Withdraws unlocked tokens from the staking contract and
+     * forwards them to an address specified by the token owner.
+     * @param receiver The receiving address.
+     * @param startFrom The start value for the iterations.
```

Comment:
```suggestion
 * @param startFrom The start date (timestamp) for the iterations.
```

Reply:
updated in this [commit](https://github.com/DistributedCollective/Sovryn-smart-contracts/pull/447/commits/79f72c867c17cd1f12ed1e726ee79e2c308a8e9e)